packages/lib-panoptes-js/package.json

```diff
@@ -14,7 +14,7 @@
   },
   "dependencies": {
     "superagent": "~7.1.1",
-    "url-parse": "~1.5.1"
+    "url-parse": "1.5.10"
```

Comment:
This pins the library to exactly 1.5.10, preventing it from being patched when new versions of 1.5.x come out. Did you intend to do that?

Reply:
Oh! No I did not intend to do that. I'll put the tilde back